.github/workflows/build-deploy.yml

```diff
@@ -50,20 +50,18 @@ jobs:
                   VAR: version=${{ env.IMAGE }}
                   RESOURCE: nais/dev-sbs.yaml
                   PRINT_PAYLOAD: true
-#
-#
 
-#deploy-to-prod:
-#   name: Deploy to prod-sbs
-#  if: github.ref == 'refs/heads/master'
-# needs: deploy-to-dev
-#runs-on: ubuntu-latest
-#     steps:
-#        - uses: actions/checkout@v1
-#       - uses: nais/deploy/actions/deploy@v1
-#        env:
-#           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
-#          CLUSTER: prod-sbs
-#         VAR: version=${{ env.IMAGE }}
-#        RESOURCE: nais/prod-sbs.yaml
-#       PRINT_PAYLOAD: true
```

Comment:
Her må det være samme indentation som `deploy-to-dev` (alt må flyttes til kolonne 5)

Reply:
navn til branch bør også endres hvor vi har denne koden: `if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/forbedring-kalkis-v2'` 
Bytt `refs/heads/forbedring-kalkis-v2` med `refs/heads/endre-tekster-og-bruke-GithubActions-for-prod`, linje 11 og 41